src/keys/rsa-browser.js

```diff
@@ -49,23 +49,21 @@ exports.unmarshalPrivateKey = function (key, callback) {
   })), callback)
 }
 
-exports.getRandomValues = function (arr) {
-  return Buffer.from(webcrypto.getRandomValues(arr))
-}
+exports.getRandomValues = randomBytes
```

Comment:
getRandomValues calls need to be fixed. You've changed the fingerprint on the param from array to number. There's also no guarantee consumers of libp2p-crypto aren't using these methods. If it's going to change, imo, we should do some type checking and then remove support for array in a future release.

Reply:
These aren't documented right? we should be ok to change. 
The only one documented is this one https://github.com/libp2p/js-libp2p-crypto#cryptorandombytesnumber. 

also rsa.js and rsa-browser.js basically feed top level randomBytes

Reply:
I think we should at least note this as a breaking change in the commit for changelog, in case someone is using it, it will be easy to see the change. The tests are currently failing because of the switch from array to number. Internal usage needs to be fixed.

Reply:
ups let me check

Reply:
please leave the array behavior intact. this is making the api less efficient than it has to be, as this means the caller is forced to allocate a new array instead of reusing an existing one

Reply:
these methods are not used nor documented, can we look into this in another issue/PR ? i can easily add support for passing arrays here https://github.com/hugomrdias/iso-random-stream/blob/master/src/random.browser.js if we need to